iptables/iptables.go

```diff
@@ -248,7 +248,12 @@ func (ipt *IPTables) runWithOutput(args []string, stdout io.Writer) error {
 	}
 
 	if err := cmd.Run(); err != nil {
-		return &Error{*(err.(*exec.ExitError)), stderr.String()}
+		switch err.(type) {
+		default:
+			return err
+		case *exec.ExitError:
```

Comment:
Do you mind switching the order of cases? It's idiomatic to have the default case as last.